crates/forge/tests/e2e/io_operations.rs

```diff
@@ -14,125 +14,110 @@ fn file_reading() {
-        [PASS] tests::test::serialization
-        [PASS] tests::test::json_serialization
-        [FAIL] tests::test::invalid_json
-        
-        Failure data:
-            Got an exception while executing a hint:
-            Parse JSON error: invalid type: integer `231232`, expected a map at line 1 column 6 , in file data/json/invalid.json
-        
-        [PASS] tests::test::json_with_array
-        [PASS] tests::test::json_deserialization
-        [PASS] tests::test::valid_content_different_folder
-        [FAIL] tests::test::non_existent
-        
-        Failure data:
-            Got an exception while executing a hint:
-            No such file or directory (os error 2)
-        
-        [FAIL] tests::test::json_non_existent
-
-        Failure data:
-            Got an exception while executing a hint:
-            No such file or directory (os error 2)
-        
-        [FAIL] tests::test::invalid_quotes
-        
-        Failure data:
-            Got an exception while executing a hint:
-            Failed to parse data/invalid_quotes.txt file
-        
-        [FAIL] tests::test::negative_number
-        
-        Failure data:
-            Got an exception while executing a hint:
-            Failed to parse data/negative_number.txt file
-        
-        [FAIL] tests::test::non_ascii
-        
-        Failure data:
-            Got an exception while executing a hint:
-            Failed to parse data/non_ascii.txt file
-        
-        [FAIL] tests::test::not_number_without_quotes
-        
-        Failure data:
-            Got an exception while executing a hint:
-            Failed to parse data/nan_without_quotes.txt file
-        
-        [FAIL] tests::test::too_large_number
-        
-        Failure data:
-            Got an exception while executing a hint:
-            Failed to parse data/too_large_number.txt file
-        
-        Tests: 9 passed, 14 failed, 0 skipped
-        
-        Failures:
-            file_reading::tests::non_existent
-            file_reading::tests::invalid_quotes
-            file_reading::tests::negative_number
-            file_reading::tests::non_ascii
-            file_reading::tests::not_number_without_quotes
-            file_reading::tests::too_large_number
-            tests::test::invalid_json
-            tests::test::non_existent
-            tests::test::json_non_existent
-            tests::test::invalid_quotes
-            tests::test::negative_number
-            tests::test::non_ascii
-            tests::test::not_number_without_quotes
-            tests::test::too_large_number
-    "#};
+    let expected = indoc! {r#"[..]Compiling[..]
+   [..]Finished[..]
+
+
+Collected 23 test(s) from file_reading package
+Running 9 test(s) from src/
+[FAIL] file_reading::tests::negative_number
+
+Failure data:
+    Got an exception while executing a hint: Hint Error: Failed to parse data/negative_number.txt file
+
+[FAIL] file_reading::tests::too_large_number
```

Comment:
Don't we want to format this as before?

Reply:
If i format this like it was formatted before it fails to match. This test is bullshit anyways, matching the output 1:1 is ridiculous if anyone asked me...